glean_parser/templates/swift.jinja2

```diff
@@ -0,0 +1,67 @@
+// -*- mode: Swift -*-
+
+// AUTOGENERATED BY glean_parser.  DO NOT EDIT.
+{# The rendered markdown is autogenerated, but this
+Jinja2 template is not. Please file bugs! #}
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+{% macro obj_declaration(obj, suffix='', access='') %}
+{{ access }}static let {{ obj.name|camelize }}{{ suffix }} = {{ obj|type_name }}(
```

Comment:
I know that I already asked for this, but I can't really find the answer. Shouldn't this be lazy?

Reply:
> Global constants and variables are always computed lazily, in a similar manner to Lazy Stored Properties. Unlike lazy stored properties, global constants and variables do not need to be marked with the lazy modifier.

That's what https://docs.swift.org/swift-book/LanguageGuide/Properties.html says.

Reply:
This makes the container `lazy` by default (e.g. `CorePing`), but does it make members of it lazy too (e.g. `seq`)?

Reply:
`CorePing` is an empty enum, it can't be instantiated (it merely acts as a namespace.

I'm pretty sure the other things are therefore "global constants" and lazily evaluated. Let me quickly verify that.

Reply:
Thanks. Sorry for being pedantic on this, but it is quite important for all the metrics to be lazy. We don't want `CorePing.seq` to be instantiated if we access `CorePing.defaultBrowser`.

Reply:
Run this test on a minimal version of that `CorePing.swift` and a bit more logging for metric instantiation:

```swift
    func testMetricLazyness() {
        print("starting lazyness test")
        let glean = Glean.shared
        glean.initialize()
        print("Glean initialized. Incremeting sequence number")
        CorePing.seq.add()
        print("CorePing.seq incremented. Next: flash usage")
        CorePing.flashUsage.add(2)
    }
```

Got this output (stripped of unnecessary information:

```
[INFO  glean_core] Creating new Glean
[INFO  glean_core::database] Database initialized
[INFO  glean_ffi] Glean initialized
Glean initialized. Incremeting sequence number
[INFO  glean_ffi::counter] Creating new metric of type 'CounterMetric' with name: seq
CorePing.seq incremented. Next: flash usage
[INFO  glean_ffi::counter] Creating new metric of type 'CounterMetric' with name: flash_usage
```

As you can see they are lazy and get created on first access.

Reply:
Great, thanks!